lib/Backend/BeeSwarm.php

```diff
@@ -69,10 +68,10 @@ public function __construct(string $appName, IL10N $l, IConfig $config, LoggerIn
 		$this->logger = $logger;
 		$this->globalStoragesService = $globalStoragesService;
 		$this
-			->setIdentifier('files_external_ethswarm')
+			->setIdentifier(AppConstants::APP_NAME)
 			->addIdentifierAlias('\OC\Files\External_Storage\BeeSwarm') // legacy compat
 			->setStorageClass('\OCA\Files_External_Ethswarm\Storage\BeeSwarm')
-			->setText($l->t('Swarm By Hejbit'))
+			->setText($l->t('Swarm'))
```

Comment:
I think @JoaoSRaposo wanted to have the HejBit in name in the settings page.

Reply:
Yes :)